src/plugins/list.js

```diff
@@ -4,53 +4,7 @@ import React from 'react';
  * This is a plugin into the markdown editor to handle lists
  */
 function List() {
-  const name = 'List';
-
-  const tags = [
-    {
-      html: 'ul',
-      slate: 'ul_list',
-      md: node => node.listType === 'bullet'
-    },
-    {
-      html: 'ol',
-      slate: 'ol_list',
-      md: node => node.listType === 'ordered'
-    },
-    {
-      html: 'li',
-      slate: 'list_item',
-      md: node => node.type === 'item'
-    }
-  ];
-
-  /**
-   * Augment the base schema with the list types
-   * @param {*} schema
-   */
-  const augmentSchema = ((schema) => {
-    const additions = {
-      blocks: {
-        ol_list: {
-          nodes: [{ match: { type: 'list_item' } }],
-        },
-        ul_list: {
-          nodes: [{ match: { type: 'list_item' } }],
-        },
-        list_item: {
-          parent: [{ type: 'ol_list' }, { type: 'ul_list' }],
-          nodes: [{ match: [{ object: 'text' }, { type: 'link' }] }],
-          marks: [{ type: 'bold' }, { type: 'italic' }, { type: 'code' }],
-        },
-      },
-    };
-
-    const newSchema = JSON.parse(JSON.stringify(schema));
-    newSchema.blocks = { ...schema.blocks, ...additions.blocks };
-    newSchema.document.nodes[0].match.push({ type: 'ol_list' });
-    newSchema.document.nodes[0].match.push({ type: 'ul_list' });
-    return newSchema;
-  });
+  const name = 'list';
```

Comment:
Is this still used? Not seeing it in this file.

Reply:
It's used by the `PluginManager` to deterministically sort the plugins so they get applied in the same order.